pybikes/data/cyclocity.json

```diff
@@ -3,17 +3,6 @@
     "class": {
         "Cyclocity": {
             "instances": [
-                {
-                    "tag": "cyclic",
-                    "meta": {
-                        "latitude": 49.44323199999999,
-                        "country": "FR",
-                        "name": "cy'clic",
-                        "longitude": 1.099971,
-                        "city": "Rouen"
-                    },
-                    "contract": "Rouen"
-                },
```

Comment:
Since there's already information on the gbfs feed, let's not remove this one yet and we can merge the PR and have both of them available until cy'clic is phased out